data/wanderers start.txt

```diff
@@ -1926,7 +1926,7 @@ mission "Wanderers Evacuation 1"
 			has "Wanderers Pond Strider Recon 2: done"
 	on offer
 		conversation
-			`Four Wanderer transports - perhaps, in fact, the same four you escorted before - are parked here. One of the captains approaches you and says, "We are evacuating the worlds that will be [isolated, trapped] if the Unfettered gain more territory. Will you escort us to <planet>, and from there to the capital?"`
```

Comment:
What's the expected timeline between the missions? Is it a reasonable duration of time w.r.t. the specialization of the ship relative to the other existing ship designs? If the timeline is rather short, maybe they have only one or two ready, and the rest become available later?

Reply:
Invaded 1: mention of the development of new warships and a dedicated transport.
Invaded 2 (2 missions later): the Tempest is revealed.
Evacuation 1 (11 missions later): the Riptide is revealed.